### PR TITLE
New community fund addresses

### PR DESCRIPTION
**Pull request by cronicc**

### CODE REVIEW

**Review by ptagl — APPROVED** [review]

File: src/gtest/test_forkmanager.cpp [hunk 1]
```diff
@@ -292,7 +421,7 @@ TEST(ForkManager, SidechainVersionForkMainnet) {
     SelectParams(CBaseChainParams::MAIN);
 
     // TODO: set proper fork height value.
```
Comment (ptagl) [inline src/gtest/test_forkmanager.cpp]: This comment can be removed.
```suggestion
```

File: src/gtest/test_forkmanager.cpp [hunk 2]
```diff
@@ -303,7 +432,7 @@ TEST(ForkManager, SidechainVersionForkTestnet) {
     SelectParams(CBaseChainParams::TESTNET);
 
     // TODO: set proper fork height value.
```
Comment (ptagl) [inline src/gtest/test_forkmanager.cpp]: This comment can be removed.
```suggestion
```

File: qa/rpc-tests/fundaddresses.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        '''
+
+        # network topology: (0)
+
+        mark_logs("Node 0 generates 500 blocks", self.nodes, DEBUG_MODE)
+        blocks = self.nodes[0].generate(500)
+        self.sync_all()
+
+        # Consider that blocks[0] is not the genesis block but the block at height 1 (second block)
+        # For this reason, all fork heights are must be decreased by 1
```
Comment (alsala) [inline qa/rpc-tests/fundaddresses.py]: syntax error in comment

**Review by alsala — APPROVED** [review]

File: src/gtest/test_forkmanager.cpp (hunk 1 above)
Comment (cronicc) [reply src/gtest/test_forkmanager.cpp]: Addressed in https://github.com/HorizenOfficial/zen/commit/6f1ffaeccf187e2903bfa608c54c5138fe7d834c

File: src/gtest/test_forkmanager.cpp (hunk 2 above)
Comment (cronicc) [reply src/gtest/test_forkmanager.cpp]: Addressed in https://github.com/HorizenOfficial/zen/commit/6f1ffaeccf187e2903bfa608c54c5138fe7d834c

File: qa/rpc-tests/fundaddresses.py (hunk 3 above)
Comment (cronicc) [reply qa/rpc-tests/fundaddresses.py]: Addressed in https://github.com/HorizenOfficial/zen/commit/6f1ffaeccf187e2903bfa608c54c5138fe7d834c

**Review by ptagl — APPROVED** [review]

**Review by albertog78 — APPROVED** [review]

**Pull request merged**